scrapy/czlscrape/utils.py

```diff
@@ -0,0 +1,15 @@
+import re
+
+
+def guess_publication_type(text, rules):
+    text = text.lower()
+    text = re.sub(r'[șş]', 's', text)
+    text = re.sub(r'[țţ]', 't', text)
+    text = re.sub(r'[ăâ]', 'a', text)
+    text = re.sub(r'[î]', 'i', text)
+
+    for substr, publication_type in rules:
```

Comment:
@margelatu De acord, o singura bucata de cod le poate acoperi pe toate. Pentru asta am scos functia respectiva in utils. Daca e nevoie de reguli speciale, se pot folosi. Daca nu, putem folosi o singura regula pentru oricate spidere.